flake-modules/dev/devshell.nix

```diff
@@ -0,0 +1,67 @@
+{inputs, ...}: {
+  imports = [
+    inputs.devshell.flakeModule
+  ];
+
+  perSystem = {
+    pkgs,
+    config,
+    system,
+    ...
+  }: {
+    devshells.default = {
+      devshell.startup.pre-commit.text = config.pre-commit.installationScript;
+
+      packages = with pkgs; [
+        python3
```

Comment:
I don't think  we should add either `python3` or `nix-output-monitor` to the packages. We should simply reference their paths in the commands (`"${lib.getExe pkgs.python3}"`).

This would risk poluting what people are using

Reply:
Done